files/custom-data/additional_userdata.sh

```diff
@@ -1,10 +1,10 @@
-# Copyright © 2021-2023, SAS Institute Inc., Cary, NC, USA. All Rights Reserved.
+# Copyright 2021-2023, SAS Institute Inc., Cary, NC, USA. All Rights Reserved.
```

Comment:
Thinking in place of the copyright symbol we may need to use (C) I believe this is an accepted legal alternative. Is there any info in the copyright guideline page?

Reply:
Yes, the guidelines here https://sasoffice365.sharepoint.com/teams/Legal/Pages/Copyright-Notices-for-SAS-Sofware.aspx indicate this:

NOTE:
Use either the word Copyright or the copyright symbol: ©

but there are three acceptable alternatives listed there using the word, the symbol, or both.